src/mixin-stache-view-test.js

```diff
@@ -103,11 +104,11 @@ if (browserSupports.customElements) {
 
 	QUnit.test("renderer is passed options object and nodelist if provided", function(assert) {
 		const options = { some: "options" };
-		const nodelist = { a: "nodelist" };
+		const nodelist = nodeLists.register([], function(){}, true, false);
 
 		const renderer = function(el, optionsArg, nodelistArg) {
 			assert.equal(optionsArg, options, "options");
-			assert.equal(nodelistArg, nodelist, "nodelist");
+			assert.deepEqual(nodelistArg, nodelist, "nodelist like object");
```

Comment:
this is sorta a hack to make the test pass ... it's no longer the nodelist you pass, instead, you get a different one.